examples/installation/nginx/nginx.conf

```diff
@@ -0,0 +1,26 @@
+http {
+  server {
+    listen 8080;
+
+    location /grafana/ {
```

Comment:
To be consistent, let's move it to additional?

Reply:
I'm not sure. Grafana is the required dependency, that why its routing is defined here.